docker-compose.yml

```diff
@@ -130,9 +130,7 @@ services:
     depends_on:
       gateway:
         condition: "service_healthy"
-      httpbin:
-        condition: "service_healthy"
-      test.httpbin.docker.local:
```

Comment:
I totally remember fixing this name after someone (maybe Andrew) pointed out that the ".docker.local" was redundant. I guess it didn't end up merged, maybe WSL lost it.

Reply:
Yeah I remember that too, not sure what happened